dev-ml/camlp5/camlp5-7.10.ebuild

```diff
@@ -0,0 +1,71 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+
+inherit multilib findlib eutils
+
+MY_P=${P%_p*}
+echo $MY_P
+DESCRIPTION="A preprocessor-pretty-printer of ocaml"
+HOMEPAGE="https://camlp5.github.io/"
+SRC_URI="https://github.com/camlp5/camlp5/archive/rel$(ver_rs 1- '').tar.gz -> ${P}.tar.gz"
+echo $SRC_URI
+S="${WORKDIR}/${PN}-rel$(ver_rs 1- '')"
```

Comment:
Please move S below RDEPEND (with one empty line in-between) so we keep good style consistency here.